### DISCUSSION TITLE
Beta GUI

### DISCUSSION THREAD

**Discussion by RFDarter** [mr-manuel/venus-os_dbus-mqtt-battery#19] [General]
hi, any planes of supporting the new beta gui?

**Comment by mr-manuel** [comment]
Yes, I‘m working with the Victron team to find a solution for all plugins, not only dbus-serialbattery.

**Comment by mr-manuel** [comment]
@RFDarter you can check this in the meanwhile. Works also for this driver: https://github.com/mr-manuel/venus-os_dbus-serialbattery/tree/dev/gui-v2

It installs in a separate directory a modded gui.
